Drafting Offline Camp featured collection

terichadbourne opened the pull request:
Here's a first stab at the wording for the main page of the Offline Camp collection (already incorporating Bradley's suggested edits from Slack. I don't have access to preview locally yet and am about to head out for vacation, so just getting this in here in case anyone needs to work on it while I'm out. 

You'll find my favorite pics and video in our Slack chat.

terichadbourne committed:
WIP Drafting Offline Camp featured collection wording

WORK IN PROGRESS - DO NOT MERGE 

Here's a first stab at the wording for the main page of the Offline Camp collection (already incorporating Bradley's suggested edits from Slack. I don't have access to preview locally yet and am about to head out for vacation, so just getting this in here in case anyone needs to work on it while I'm out. 

You'll find my favorite pics and video in our Slack chat.

terichadbourne committed:
Removed loopy thing

terichadbourne committed:
Removing SXSW podcast from Offline Camp collection

This one is not related to camp, so I changed removed the features: offline-camp and instead put strategies: offline-first

terichadbourne committed:
Adding projects to Offline Camp collection

Crafting Community Events that Leave them Wanting S'more - Medium post on event format
Offline Camp Experience - video
Offline Camp California -  video
Offline Camp Berlin - video
Cali camp recap w/ sessions & passion talks linked to - Medium
Berlin camp recap w/ sessions & passion talks linked to - Medium
Catskills to Cali post w/ sessions & passion talks from Catskills - Medium
Offline First Podcasts - curated list of all we know of, including those from camp - Medium (also linked directly to OF strategy)

terichadbourne commented:
@bradnoble @bradley-holt @mikebroberg I've now also added some key camp-related projects to get us started. There are more that can be chosen from our Box doc as needed, and I can take a closer look at that when I'm back in the office.  Please feel free to edit and incorporate as much of this as you'd like while I'm out, and I'm happy to do more when back.

bradley-holt commented on _data/projects.yml:
```diff
@@ -361,4 +361,72 @@
\ No newline at end of file
+    - Cloudant
+    
+ - title:    Crafting Community Events that Leave them Wanting S'more
```
This is invalid YAML. There should be no space before the `-` here.

bradley-holt commented on _data/projects.yml:
```diff
@@ -361,4 +361,72 @@
+  features: 
+    - offline-camp    
+    
+ - title:   Offline Camp California 2016
```
This is invalid YAML. There should be no space before the `-` here.

bradley-holt commented on _data/projects.yml:
```diff
@@ -361,4 +361,72 @@
+    - offline-camp    
+
+    
+- title:   Recap: Offline Camp Berlin
```
This is invalid YAML. The value here should be in quotes as it contains a colon.

bradley-holt commented on _data/projects.yml:
```diff
@@ -361,4 +361,72 @@
+  features: 
+    - offline-camp    
+    
+- title:   Recap: Offline Camp California
```
This is invalid YAML. The value here should be in quotes as it contains a colon.

bradley-holt commented on _data/projects.yml:
```diff
@@ -361,4 +361,72 @@
+  features: 
+    - offline-camp    
+   
+- title:   Recap: Offline Camp Catskills
```
This is invalid YAML. The value here should be in quotes as it contains a colon.

bradley-holt reviewed (dismissed):
I have noted several instances of invalid YAML in `_data/projects.yml`. These errors are causing the page build to fail.

terichadbourne commented:
Ooh, thanks for the formatting catches @bradley-holt! I'm hoping to do more work on this section this week, which I assume will require me resolving some conflicts with your branch, so I can then try to figure out the "resolve conflicts" need mentioned above.

Commit:
corrected formatting as suggested by Bradley

terichadbourne commented:
@bradley-holt I made those edits to the .yml file and it's now building correctly in Jekyll so I can preview my own edits on this branch. 

Do you feel done with your Offline First branch? Since our things are so related, if you feel you're done, I will merge yours into mine (which won't affect yours or the master) so that I can be seeing the end result of our things combined when I work locally.

bradley-holt commented on _features/offline-camp.md:
```diff
@@ -9,31 +9,25 @@ img-thumb:  offline-camp-campfire.jpeg #offlinecamp-logo-thumb.jpg
 ---
 
-this is the first paragraph this is the first paragraph this is the first paragraph this is the first paragraph this is the first paragraph. this is the first paragraph this is the first paragraph this is the first paragraph this is the first paragraph this is the first paragraph.
+Offline First is about more than just technology; it’s about community. That’s why we’re so happy to be foundational sponsors of the community-driven unconference known as [Offline Camp](http://www.offlinefirst.org/camp). Each retreat takes place over the course of a long weekend in a remote location, gathering developers, UX professionals, and business leaders to discuss the opportunities and ch ...
```
I believe the canonical URL for Offline Camp is `http://offlinefirst.org/camp/` (rather than `http://www.offlinefirst.org/camp`).

bradley-holt replied:
Same with the Offline First URL, which should be `http://offlinefirst.org/` (rather than `http://www.offlinefirst.org`).

bradley-holt commented on _features/offline-camp.md:
```diff
@@ -9,31 +9,25 @@ img-thumb:  offline-camp-campfire.jpeg #offlinecamp-logo-thumb.jpg
 ---
 
-this is the first paragraph this is the first paragraph this is the first paragraph this is the first paragraph this is the first paragraph. this is the first paragraph this is the first paragraph this is the first paragraph this is the first paragraph this is the first paragraph.
+Offline First is about more than just technology; it’s about community. That’s why we’re so happy to be foundational sponsors of the community-driven unconference known as [Offline Camp](http://www.offlinefirst.org/camp). Each retreat takes place over the course of a long weekend in a remote location, gathering developers, UX professionals, and business leaders to discuss the opportunities and ch ...
```
Where you say, "Our team has sponsored…" I would consider instead saying something like, "The IBM Watson Data Lab team has sponsored…"

terichadbourne committed:
Corrected URLs for offlinefirst.org sites

Based on Bradley's review, I've fixed the URLs to remove the "www."

terichadbourne commented:
Fixed those two URLs per @bradley-holt's review. And now on to the pics / videos / Twitter embeds!

Commit:
Added video and Twitter embeds

Commit:
Reordered Twitter embeds

terichadbourne commented:
I've added the visuals we need for the collection, in the form of video and Twitter embeds (there's so much already that I'm not adding an image).

I did my "Reordered Twitter embeds" commit after finding that Mikeal's Twitter Moment was so long that someone might never make it to my last paragraph.

Commit:
Added iframe max width to site CSS

Commit:
Added Offline First Resources to essentials

Commit:
Added Offline First slack team to essentials

Commit:
removed Google sponsorship project from OF strategy page

Commit:
reordered Offline Camp projects

Commit:
Reversed order of projects to correct display

Commit:
Consolidating Medium and YouTube for each event

Joined what were previously two separate projects per event -- one Medium recap (including list of sessions covered that link out to those posts) and one YouTube video recap -- into a single project with multiple links.

Commit:
Combining camp experience article and video

Joined the Medium article and YouTube video into a single project explaining the uniqueness of the Offline Camp experience.

terichadbourne commented:
After fixing the technical glitches @bradley-holt caught in his review, I ended up consolidating what used to be 2 projects per event (one Medium and one YouTube) into a single project since the descriptions were already quite similar and it looked like too much content for the page layout previously.  (As a reminder, there is no video in existence for the Catksills.)

I did the same for the "Crafting Community Events That Leave Them Wanting S'more" article and the "Offline Camp Experience Video" (that video is also now embedded on the collection page, as are a couple of key Tweets with positive feedback). 

I also fixed the ordering in the yml file (per @vabarbosa's guidance) to make things to display in what I feel is the appropriate order (overall experience, then events most recent first, then odds & ends.

I'm can now  preview on my laptop and am pretty comfortable that this is ready to publish, but would welcome any suggestions from @mikebroberg on improving titles or headlines of the projects.

I do have more ideas for content that we could include here and/or on the Offline First strategy page, as discussed with @bradley-holt @bradnoble @mikebroberg back before my vacation, but many of those ideas would be much easier to implement if we could see Bradley's strategy branch and my collections branch merged and live first. My assumption is that that's something Broberg would be responsible for doing after he's back from vacation and has had a chance to dig out.

mikebroberg committed:
Edit offline camp featured collection

I only broke up two paragraphs. I added a short sentence to set up the
Twitter memory at the end. Otherwise, no changes or typos.

mikebroberg committed:
Merge branch 'master' into offline-camp-collection

# Conflicts:
#	_data/projects.yml
#	_features/offline-camp.md

mikebroberg committed:
Minor text fix to Offline Camp to set up Twitter moment
